pkgs/top-level/all-packages.nix

```
@@ -756,6 +756,8 @@ in
 
   git-fire = callPackage ../tools/misc/git-fire { };
 
+  git-repo-updater = python37Packages.callPackage ../development/tools/git-repo-updater { };
```

```suggestion
  git-repo-updater = python3Packages.callPackage ../development/tools/git-repo-updater { };
```
Actually it's probably better to just say `python3Packages` because it works with all Python3 Versions. Otherwise we'd probably forget updating it when a new Python version comes out. `python3Packages` is `python37Packages` anyway.